releases/release-1.20/release_team.md

```diff
@@ -8,7 +8,7 @@
 | Bug Triage | Vlad Gorodetsky ([@bai](https://github.com/bai) / Slack: `@bai`) | Max Körbächer ([@mkorbi](https://github.com/mkorbi) / Slack: `@mkorbi`), Nikolas Moraitis ([@droslean](https://github.com/droslean) / Slack: `@droslean`), Menna Elmasry ([@MonzElmasry](https://github.com/MonzElmasry) / Slack: `@Menna`), Derrik Campau ([@erismaster](https://github.com/erismaster) / Slack: `@dcampau`), Sayan Chowdhury ([@sayanchowdhury](https://github.com/sayanchowdhury) / Slack: `@yudocaa`) |
 | Docs | Anna Jung ([@annajung](https://github.com/annajung) / Slack: `@annajung`) | Leslie-Alexandre Denis ([@eagleusb](https://github.com/eagleusb) / Slack: `@grumpycat`), Rey Lejano ([@reylejano-rxm](https://github.com/reylejano-rxm) / Slack: `@rey.lejano`), Kristin Martin ([@kcmartin](https://github.com/kcmartin) / Slack: `@kcmartin`), Somtochi Onyekwere ([@SomtochiAma](https://github.com/SomtochiAma) / Slack: `@SomtochiAma`)  |
 | Release Notes | James Laverack ([@JamesLaverack](https://github.com/JamesLaverack) / Slack: `@james.laverack`) | |
-| Communications | Joseph Sandoval ([@jrsapi](https://github.com/jrsapi) / Slack: `@j-dawg`) | |
+| Communications | Joseph Sandoval ([@jrsapi](https://github.com/jrsapi) / Slack: `@j-dawg`) | Divya Mohan ([@divya-mohan0209](https://github.com/divya-mohan0209) /  Slack: @divya), Shubham Tatvamas ([@shubhamtatvamasi](https://github.com/ShubhamTatvamasi) / Slack: @Shubham Tatvamasi ), Tony Gosselin ([@sfotony](https://github.com/sfotony) / Slack: @sfotony ), Chris Short [(@chris-short)](https://github.com/chris-short) / @chrisshort ) | |
```

Comment:
nitpick: I think we usually list alphabetically by last name :)

Reply:
i'll reorder by last name.